use_cases/details/relational-db.md

```diff
@@ -1,13 +1,13 @@
-Gretel Relational takes you beyond synthesizing and transforming one table at a time, and enables generating entire databases, all while maintaining referential integrity. This early preview will give you a taste of what's coming in the Gretel Console and CLI/SDK. We'd love to hear your feedback!
+Gretel Relational takes you beyond synthesizing and transforming one table at a time, and enables generating entire databases, all while maintaining referential integrity. This early preview will give you a taste of what's coming in the Gretel Console and CLI/SDK. [We'd love to hear your feedback!](https://dqq4jigtkl1.typeform.com/to/Gibb8awJ)
 
 ### Notebook Details
 
 In the notebooks below, we've created two practical examples using a telecommunications database and Gretel Relational. See the database schema below.
 
-![telecom db diagram](https://blueprints-dev.gretel.cloud/use_cases/images/telecom-db-small.png "Telecom Database Diagram")
+![telecom db diagram](https://blueprints.gretel.cloud/use_cases/images/telecom-db-small.png "Telecom Database Diagram")
```

Comment:
Something funky is causing:
<img width="1138" alt="Screenshot 2023-03-22 at 1 20 18 PM" src="https://user-images.githubusercontent.com/52251896/227028650-f47c9aed-1f30-4257-8377-7dfc0c83fa97.png">

Should we try using html within md instead?

Reply:
chatted in slack